flare/bin/flare.py

```diff
@@ -11,27 +10,55 @@
 APP_NAME = "flare"
 
 
+def get_flare_api_client(
+    *,
+    api_key: str,
+    tenant_id: t.Union[int, None],
+) -> FlareApiClient:
+    api_client = FlareApiClient(
+        api_key=api_key,
+        tenant_id=tenant_id,
+    )
+    current_user_agent: t.Union[str, bytes] = (
+        api_client._session.headers.get(
+            "User-Agent",
+        )
+        or ""
+    )
+    if isinstance(current_user_agent, bytes):
+        current_user_agent = current_user_agent.decode()
+    api_client._session["User-Agent"] = f"{current_user_agent} flare-splunk"
```

Comment:
It would be even better in a follow-up PR if we can use `flare-splunk/{version}`. Where `version` would be the  version of the app.

Reply:
The app is broken here. Seeing these messages

`TypeError: 'Session' object does not support item assignment`
`api_client._session["User-Agent"] = f"{current_user_agent} flare-splunk".strip()`

Reply:
oops. this should be `api_client._session.headers["User-Agent"]`

Reply:
> This PR is completely untested. So it would be helpful if one of you can test it. Or feel free to just integrate my suggested changes yourself in separate commits.

Ah, just saw this. OK, will do. Thanks.